src/field/pokemon.ts

```diff
@@ -4147,6 +4150,14 @@ export class PlayerPokemon extends Pokemon {
     return this.getFieldIndex();
   }
 
+  override getAlly(): Pokemon | undefined {
+    const ally = globalScene.getPlayerField()[this.getFieldIndex() ? 0 : 1];
```

Comment:
```suggestion
    const ally = super.getAlly();
```
Technically it's the same but in case the logic behind Pokemon.getAlly ever gets changed it ensures it would get reflected here